sdk/core/core-lro/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0-preview.2 - 2019-10-24
```

Comment:
Nit: Can the date be TBD until we know when we'll release it? That helps me quickly tell if it's been released already or not :)

Reply:
addressed!